_topic_maps/_topic_map_osd.yml

```
@@ -249,6 +249,81 @@ Topics:
     - Name: Configuring custom domains for applications
       File: osd-config-custom-domains-applications
 ---
+Name: Logging
+Dir: logging
+Distros: openshift-osd
```

```suggestion
Distros: openshift-dedicated
```
This is true throughout.